usagov/usagov-2021
# Usagov 1567 accordion hover

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1567

## Description

Add the hover and down/open states back onto the accordions

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- This includes all grey accordions, including Directories, CEO tool, branches of government.
- Goto http://localhost/agency-index, http://localhost/branches-of-government, http://localhost/elected-officials (accordions on the results page)
- Verify the hover, open state of accordion and focus color on the accordions
             Regular state: #F0F0F0
             Hover state: #C9C9C9
<img width="1512" alt="Screenshot 2024-05-23 at 1 19 31 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/086ae926-d09f-4f76-8ef6-4dbe2dbda740">

<img width="1512" alt="Screenshot 2024-05-23 at 1 19 47 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/9768ccd7-ad6d-4945-87ff-16da66853f67">

<img width="1512" alt="Screenshot 2024-05-23 at 1 20 14 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/26f7bbb0-c852-4bc9-be54-80d0f01557d6">



             Down/Open state: #C9C9C9 with 2px focus stroke inner #FFFFFF and outer 2px #0186a2
-Verify on the sub accordions on the http://localhost/elected-officials results page 
             Regular state: # CCECF2
             Hover state: #99DEEA
             Down/Open state: #99DEEA with focus stroke inner 2px #FFFFFF and outer 2px # 0186a2
<img width="1512" alt="Screenshot 2024-05-23 at 1 18 37 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/3fa63307-8c73-4588-beb1-b6117b2b7fd3">



## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
